README.md

```diff
@@ -6,9 +6,7 @@
 
```

Comment:
@sabineschaller this clock change stuff always messes with me, but is 1100 GMT still correct? I believe if the meeting is 1300 CET, then it should be 1200GMT